Source/TargetGizmo.cs

```diff
@@ -93,7 +93,8 @@ public static PurposeInfo Make(LocalTargetInfo targetInfo)
 						}
 						else
 						{
-							purposeInfo.icon = PortraitsCache.Get(pawn, Vector2.one * Gizmo.Height, default(Vector3), 1.5f);
+							purposeInfo.icon = PortraitsCache.Get(pawn, Vector2.one * Gizmo.Height, default, default(Vector3), 1.5f, true, true, false, true, null, null, false);
```

Comment:
For some reason default no longer faces the pawn portrait south. It should be this instead. My bad! 
`purposeInfo.icon = PortraitsCache.Get(pawn, Vector2.one * Gizmo.Height, default, default(Vector3), 1.5f, true, true, false, true, null, null, false);`